README.md

```diff
@@ -30,17 +30,25 @@ Follow up instructions located [here](https://github.com/MoveOnOrg/Spoke/blob/ma
 6. Create an [Auth0](https://auth0.com) account. In your Auth0 account, go to Settings -> Clients -> and then grab your Client ID, Client Secret, and your Auth0 domain (should look like xxx.auth0.com). Add those inside your `.env` file (AUTH0_CLIENT_ID, AUTH0_CLIENT_SECRET, AUTH0_DOMAIN respectively).
 7. Run `npm run dev` to create and populate the tables.
 8. Add the login callback and logout callback URL in `.env` (default `http://localhost:3000/login-callback` and `http://localhost:3000/logout-callback`) to your Auth0 app settings under "Allowed Callback URLs" and "Allowed Logout URLs" respectively. (If you get an error when logging in later about "OIDC", go to Advanced Settings section, and then OAuth, and turn off 'OIDC Conformant')
-9. Run `npm run dev` to start the app. Wait until you see both "Node app is running ..." and "Webpack dev server is now running ..." before attempting to connect. (make sure environment variable `JOBS_SAME_PROCESS=1`)
-10. Go to `http://localhost:3000` to load the app.
-11. As long as you leave `SUPPRESS_SELF_INVITE=` blank and unset in your `.env` you should be able to invite yourself from the homepage.
+9. Add the domain name for your app to "Allowed Web Origins" in the Auth0 Settings. It should be the same as the login/logout callback URLs without the callback path. So for localhost it would look like "http://localhost:3000"
+10. Add a new [rule](https://manage.auth0.com/#/rules/create) in Auth0: 
```

Comment:
Can we add something more to ten to make it super clear? I used the 'empty' rule prompt, and then after it was successfully created, was able to test using: https://manage.auth0.com/#/rules and clicking on 'try all rules with ... (username-password-authentication)'. I'm not sure if it's worth adding testing instructions.